.gitignore

```diff
@@ -18,4 +18,5 @@ out/*/*
 !out/Greeter.sol/*
 
 # Keep the latest deployment only
-broadcast/*/*/*
+broadcast/**
+!broadcast/**/run-latest.json
```

Comment:
Do we really want to push all of this runs?

Reply:
Yes, we want to keep track of all the deployments, Tagging @just-a-node here for clarity.

Reply:
Yeah, mainly for proof of deployment and contract address. Is there a concern in exposing the broadcasts?